src/astro/databases/base.py

```diff
@@ -158,7 +159,7 @@ def populate_table_metadata(self, table: Table) -> Table:
         if table.metadata and table.metadata.is_empty() and self.default_metadata:
```

Comment:
```
if table.metadata and table.metadata.is_empty() and self.default_metadata:  ## ---- 1
    table.metadata = self.default_metadata
if not table.metadata.schema:    ## ---- 2
    table.metadata.schema = self.DEFAULT_SCHEMA
```
@tatiana I think this scenario should be covered by the existing code. 

When table don't have metadata, will be assigned by 1.
Then we check if table has Schema or not if not we assign it from self.DEFAULT_SCHEMA -- 2

Also, self.DEFAULT_SCHEMA will always have value, because we use code(setting.py)
`SCHEMA = conf.get("astro_sdk", "sql_schema", fallback=DEFAULT_SCHEMA)`
Where `DEFAULT_SCHEMA` is a constant.

Reply:
Makes sense, @utkarsharma2 !